experimental/unitsconversion/tests/units_test.rs

```diff
@@ -76,9 +77,12 @@ fn test_conversion() {
         })
         .collect();
 
-    let parser = MeasureUnitParser::from_payload(
+    // TODO: how to convert from `&ZeroTrie<ZeroVec<'_, u8>>` to &ZeroTrieSimpleAscii<Vec<u8>>?
+    let payload: ZeroTrieSimpleAscii<Vec<u8>> = ZeroTrieSimpleAscii::try_from(
         &icu_unitsconversion::provider::Baked::SINGLETON_UNITS_INFO_V1.units_conversion_trie,
-    );
+    )
+    .unwrap();
+    let parser = MeasureUnitParser::from_payload(&payload);
 
```

Comment:
@sffc :  how to convert from `&ZeroTrie<ZeroVec<'_, u8>>` to &ZeroTrieSimpleAscii<Vec<u8>>?

Or more accurate question: how to get the `ZeroTrieSimpleAsciiCursor` from `ZeroTrie<ZeroVec<'_, u8>>`

Reply:
For now use take_store and from_store

After #4408 we can simplify it

Reply:
I have done that.

However, `take_store` returns `ZeroVec<u8>` and `from_store` need `Vec<u8>`\

is there a better way than converting `ZeroVec` to `Vec`?